From: Patrick Ruddy patrick.ruddy@intl.att.com>
Date: Mon, 30 Sep 2019 15:59:07 +0100
Subject: SNMP: add context name to findVar

vyatta nrt-snmp version for findVar has an extra context name
parameter. Add it here to avoid build issues.

---
 lib/agentx.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/agentx.c
+++ b/lib/agentx.c
@@ -313,7 +313,7 @@ int smux_trap(struct variable *vp, size_
 			cvp.findVar = vp[j].findVar;
 			/* Grab the result. */
 			val = cvp.findVar(&cvp, oid, &oid_len, 1, &val_len,
-					  &wm);
+					  &wm, "");
 			if (!val)
 				break;
 			snmp_varlist_add_variable(&notification_vars, oid,
